templates/case-studies/templates/_macros-text.jinja

```diff
@@ -0,0 +1,15 @@
+# Params
+# title: Title of text section (required)
+# description: Description of text section (required)
+
+{%- macro text(title, description) -%}
+  <section class="p-strip is-deep">
+    <div class="row">
```

Comment:
```suggestion
    <div class="row p-section--shallow">
```
Looks like this should be wrapped as well 
![image](https://github.com/user-attachments/assets/40fae673-738a-45ea-99f2-e1597cc04c49)